communitiesuk/funding-service-design-workflows
# FS-2563: Send email for incomplete applications

Ticket : https://dluhcdigital.atlassian.net/jira/software/c/projects/FS/boards/50?selectedIssue=FS-2563

The ticket above describes the requirement to send a copy of any incomplete applications. This is a type of notification, similar to the existing 'deadline reminder' notification that currently exists. We now have multiple notifications that need to be managed (and perhaps more in future). More generally, these can be considered as types of 'events' that need to be processed during fund rounds. This PR adds two functions, one to pull events from the fund store and hand them over to the relevant processor, and another (a processor) to send applicants a copy of their incomplete applications.  

Change description
A brief description of the pull request

[N/A] Unit tests and other appropriate tests added or updated
[N/A] README and other documentation has been updated / added (if needed)
[Done] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")
How to test
QA testing will require creation of an incomplete application for a fund round, and a dev to create an (active) event in the fund store. The lambda function should pull the event and send an email for the incomplete application. 

Screenshots of UI changes (if applicable)
N/A
